spec/features/user_register_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+feature "registration" do
+  scenario "user creates account and logs in" do
+    register_alice
+    visit sign_in_path
+    fill_in "email", with: "alice@example.com"
+    fill_in "password", with: "password"
+    click_button "Sign In"
+    expect(page).to have_content("Welcome, #{}")
```

Comment:
You don't need the empty interpolation here. It seems you may have intended to put a name there?